src/extension.ts

```diff
@@ -66,6 +66,7 @@ const COMMAND_ID_OPEN_OPERATOR_LOG = 'camelk.integrations.openOperatorLog';
 export const COMMAND_ID_START_JAVA_DEBUG = 'camelk.integrations.debug.java';
 export async function activate(context: vscode.ExtensionContext) {
 	stashedContext = context;
+	telemetry.initializeTelemetry(context);
```

Comment:
try to await here?

Reply:
still same error I do not think that it is not a problem of timing. When I tried initially, i put traces and they were written in correct order.